tests/conftest.py

```diff
@@ -946,19 +946,22 @@ def tier_marks_name():
 
 @pytest.fixture(scope='function', autouse=True)
 def health_checker(request, tier_marks_name):
+    skipped = False
```

Comment:
do we need this boolean flag? can you explain how this is used?

Reply:
In case the test marked to skip, you don't want to run health check in the teardown

Reply:
shoudn't we get rid of complete finalizer then? Since that path can never be reached now?

Reply:
I see the difference, resolving, Thx